recipes/libmodbus/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE libmodbus::libmodbus)
```
CONAN_LIBS is no longer used and will be deprecated.